package.json

```diff
@@ -76,10 +76,10 @@
     "cordova-android": "~9.0.0",
     "cordova-browser": "~6.0.0",
     "cordova-ios": "~6.1.0",
-    "cordova-osx": "~6.0.0",
-    "cordova-plugin-clipboard": "Jigsaw-Code/outline-cordova-plugin-clipboard#v2.0.0",
+    "cordova-osx": "github:Jigsaw-Code/cordova-osx",
```

Comment:
I think it's fine to keep like this because package-lock.json will pin the commit.